recipes/libosmium/all/conanfile.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
+
+class LibosmiumConan(ConanFile):
+    name = "libosmium"
+    description = "A fast and flexible C++ library for working with OpenStreetMap data"
+    license = "BSL-1.0"
+    homepage = "https://osmcode.org/libosmium/"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("openstreetmap", "osm", "basemap", "gis", "geography", "header-only")
+
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "pbf": [True, False],
+        "xml": [True, False],
+        "geos": [True, False],
+        "gdal": [True, False],
+        "proj": [True, False],
+        "lz4": [True, False],
+    }
+    default_options = {
+        "pbf": True,
+        "xml": True,
+        "geos": True,
+        "gdal": False,
+        "proj": True,
+        "lz4": True,
+    }
+    options_description = {
+        # https://github.com/osmcode/libosmium/blob/v2.20.0/cmake/FindOsmium.cmake#L30-L37
+        "pbf": "include libraries needed for PBF input and output",
+        "xml": "include libraries needed for XML input and output",
+        "geos": "include if you want to use any of the GEOS functions",
+        "gdal": "include if you want to use any of the OGR functions",
+        "proj": "include if you want to use any of the Proj.4 functions",
+        "lz4": "include support for LZ4 compression of PBF files",
+    }
+
+    def export_sources(self):
+        copy(self, "libosmium-official-vars.cmake", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.pbf:
+            self.requires("protozero/1.7.1")
+        if self.options.xml:
+            self.requires("expat/[>=2.6.2 <3]")
+            self.requires("bzip2/1.0.8")
+        if self.options.pbf or self.options.xml:
+            self.requires("zlib/[>=1.2.11 <2]")
+        if self.options.geos:
+            self.requires("geos/3.12.0")
+        if self.options.gdal:
+            self.requires("gdalcpp/1.3.0")
```

Comment:
is this a typo @valgur? I don't see any `gdalcpp` recipe

Reply:
I just didn't remember, this PR needs https://github.com/conan-io/conan-center-index/pull/23829 for this option to be true-able!